doc/languages-frameworks/python.section.md

```diff
@@ -1514,11 +1514,11 @@ Note: There is a boolean value `lib.inNixShell` set to `true` if nix-shell is in
 
 ### Tools {#tools}
 
-Packages inside nixpkgs are written by hand. However many tools exist in
-community to help save time. No tool is preferred at the moment.
+Packages inside nixpkgs must use the `buildPythonPackage` or `buildPythonApplication` function,
```

Comment:
This wording isn't really good enough as tooling like poetry2nix is _also_ using these functions to create it's packages.